pyanaconda/modules/storage/dasd/dasd.py

```diff
@@ -21,20 +21,72 @@
 from pyanaconda.dbus import DBus
 from pyanaconda.modules.common.base import KickstartBaseModule
 from pyanaconda.modules.common.constants.objects import DASD
+from pyanaconda.modules.common.errors.storage import UnavailableStorageError, UnknownDeviceError
 from pyanaconda.modules.storage.dasd.dasd_interface import DASDInterface
 from pyanaconda.modules.storage.dasd.discover import DASDDiscoverTask
-from pyanaconda.modules.storage.dasd.format import DASDFormatTask
+from pyanaconda.modules.storage.dasd.format import DASDFormatTask, FindFormattableDASDTask
 
 log = get_module_logger(__name__)
 
 
 class DASDModule(KickstartBaseModule):
     """The DASD module."""
 
+    def __init__(self):
+        super().__init__()
+        self._storage = None
+        self._can_format_unformatted = False
+        self._can_format_ldl = False
+
     def publish(self):
         """Publish the module."""
         DBus.publish_object(DASD.object_path, DASDInterface(self))
 
+    @property
+    def storage(self):
+        """The storage model.
+
+        :return: an instance of Blivet
```

Comment:
Please document also the exception here.

Reply:
Fixed.